docs/state-store-overview.md

```diff
@@ -34,6 +42,37 @@ Streaming state store has distinguished workload characteristics.
 
 This leads to the design of Hummock, the cloud-native KV-based streaming state store. We’ll explain concepts like “epoch”, “key space” and “barrier” in the following chapters.
 
+## Relational Table
+[source code](https://github.com/singularity-data/risingwave/blob/main/src/storage/src/table/state_table.rs)
+
+In this part, we will introduce how stateful executors interact with KV state store through the relational table layer.
+
+Relational table layer consists of State Table, Mem Table and Cell-based Table. The State Table provides the table operations by these APIs: `get_row`, `scan`, `insert_row`, `delete_row` and `update_row`, which are the read and write  interfaces for executors. The Mem Table is an in-memory buffer for caching table operations during one epoch, and the Cell-based Table is responsible for performing serialization and deserialization between cell-based encoding and KV encoding.
+
+![Overview of Relational Table](images/relational-table-layer/relational-table-01.svg)
```

Comment:
State Table / Cell-based Table in this part looks confusing. I'd prefer to call the rectangle wrapped part "Relational Table Layer". No need to mention there's memtable and "Cell-based Table". You can describe this part later in read / write path.

Reply:
State Store Layer -> KV State Store Layer